web-components/src/components/table-advanced/tokens/md-table-advanced-tokens.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-undef */
+const colors = require("@momentum-ui/tokens/dist/colors.json");
+
+const tableAdvanced = {
+  prefix: "md",
+  component: "table-advanced",
+  header: {
+    "bg-color": {
+      light: colors.gray["05"].name,
+      dark: colors.gray[95].name
```

Comment:
Yes, that's right, I know that. But I noticed one thing, if we use global variables and don't use the above theme container, they don't apply. And since this table is a very complex component that can possibly be used in different interfaces, I added this as internal tokens so that it does not depend on the theme container.